with-tv/app.json

```diff
@@ -13,7 +13,8 @@
             "topShelf2x": "./assets/images/icon-3840x1440.png",
             "topShelfWide": "./assets/images/icon-2320x720.png",
             "topShelfWide2x": "./assets/images/icon-4640x1440.png"
-          }
+          },
+          "isTV": true
```

Comment:
This example is configured to be able to build either TV or mobile apps, depending on the value of the `EXPO_TV` environment variable. (See the [config-tv plugin usage section](https://github.com/react-native-tvos/config-tv/blob/main/packages/config-tv/README.md#usage) of its README.

Making this change would break that flexibility, and also require changes in the README of this example, and also require changes in the `with-router-tv` example so that they work the same way. We would also have to modify the [Expo docs for TV](https://docs.expo.dev/guides/building-for-tv/) since they expect the `EXPO_TV` environment variable to be used to select TV builds.

Reply:
Closing this PR for now... @anishamalde I'm happy to discuss this in the [`#tv` channel of the Expo Discord server](https://discord.com/channels/695411232856997968/1069691536201224304) if you have suggestions for changes like this.